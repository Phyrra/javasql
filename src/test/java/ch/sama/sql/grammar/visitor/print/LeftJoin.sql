select *
from t1
left join t2 on t2.key = t1.key