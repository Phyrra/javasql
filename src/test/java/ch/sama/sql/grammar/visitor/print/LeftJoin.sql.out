sql
	statement
		select
			value
				all
		from
			source
				table
					t1
		join
			left
			source
				table
					t2
			condition
				primary
					table field
						t2
						key
				primary
					table field
						t1
						key
